data/alignments/eng/ESV/SBLGNT-ESV-manual.toml

```diff
@@ -21,7 +21,7 @@ license = "CC-BY 4.0"
 
 [alignment]
 
-identifier = "NA27-ESV-manual"
+identifier = "SBLGNT-ESV-manual"
 format = "grapecity"
 license = "CC-BY-4.0"
 process = "manual"
```

Comment:
@sboisen this is the second commit where I addressed that problem (after I noticed it, totally missed it the first time).